### ISSUE TITLE
Missing reference

### ISSUE THREAD

**Issue by mjuskiewicz** [oznetmaster/SSharpReflectionExtensionsLibrary#1]
Hello, I would like to compile your code but I cannot find reference to 
<HintPath>..\..\..\Simpl# Pro Libraries\SSharpNUnitLite3\SSharpNunitLite\bin\Release\SSharpNunitLite.dll</HintPath>

Could tell me where it is available?

**Comment by oznetmaster** [comment]
That is only needed if you are compiling the unit tests.

 

If you are doing unit testing, then you would have the entire NUnit dependencies on your computer already.

**Comment by mjuskiewicz** [comment]
I added the original question because in my internal project I use Crestron libs:
SimplSharpCustomAttributesInterface
SimplSharpHelperInterface
SimplSharpReflectionInterface
Code works fine and can be compiled. Unfortunately when I run a unit tests it always shows a error message that some dll's are missing: System.IO.FileNotFoundException: Could not load file or assembly 'ManagedUtilities, Version=1.0.7188.14342 . I was wondering how do you resole the issue? The problem occurs when I test a code that use for example ErrorLog class or Crestron reflection.

**Comment by oznetmaster** [comment]
Is your internal project a Simpl# Library?

This error implies you have not set up your compilation environment correctly.

**Comment by mjuskiewicz** [comment]
Yes, It is a Simpl# library. As I said, the code works fine on the processor but I cannot run a unit tests for all components. Could you give me some tips ? I will be very grateful
